Development/nmos/test/sdp_test_utils.h

```diff
@@ -0,0 +1,14 @@
+#ifndef NMOS_SDP_TEST_UTILS_H
+#define NMOS_SDP_TEST_UTILS_H
+
+namespace nmos
+{
+    struct sdp_parameters;
+}
+
+namespace sdp_test
```

Comment:
I think this should be within `nmos` namespace probably, since top-level namespace `sdp_test` might be used by sdp module.